### ISSUE TITLE
Unknown Variable `middle-1` in the `ArrayMedian` Function

### ISSUE THREAD

**Issue by briantsaunders** [Sett17/typst-statastic#2]
When calling the `arrayMedian` function and only passing three values in the array it throws an error saying `unknown variable: middle-1`.

```typst
#import "@preview/statastic:0.1.0": arrayMedian

#let data = (46, 93, 360)
#let median = arrayMedian(data)
``` 

Changing `col.at(middle-1)` to `col.at(middle - 1)` appears to fix the issue.

**Comment by dikkadev** [comment]
Great catch! Would you be open to create a PR with that?

**Comment by dikkadev** [comment]
This is fixed in d525748d4a3877fffde27562d8a10b5652c17e43
and will be in version 1.0.0 when that is merged https://github.com/typst/packages/pull/1158
